gradle/libs.versions.toml

```
@@ -1,7 +1,7 @@
 [versions]
 dokka = "1.6.21"
 jacoco = "0.8.8"
-kotlin = "1.6.21"
+kotlin = "1.7.0"
```

Should be 1.7.10 then.

```suggestion
kotlin = "1.7.10"
```